jekyll/_cci2/building-docker-images.md

```diff
@@ -118,7 +118,30 @@ If your job requires a specific docker version, you can set it as a `version` at
           version: 18.06.0-ce
 ```
 
-CircleCI supports multiple versions of Docker and defaults to using `docker-17.09.0-ce`. Consult the [Stable releases](https://download.docker.com/linux/static/stable/x86_64/) or [Edge releases](https://download.docker.com/linux/static/edge/x86_64/) for the full list of supported versions.
+CircleCI supports multiple versions of Docker and defaults to using `17.09.0-ce`. Following are the supported stable and edge versions:
+
+- `17.03.0-ce`
+- `17.05.0-ce`
+- `17.06.0-ce`
+- `17.06.1-ce`
+- `d17.07.0-ce`
+- `17.09.0-ce`
+- `17.10.0-ce`
+- `17.11.0-ce`
+- `17.12.0-ce`
+- `17.12.1-ce`
+- `18.01.0-ce`
+- `18.02.0-ce`
+- `18.03.0-ce`
+- `18.03.1-ce`
+- `18.04.0-ce`
+- `18.05.0-ce`
+- `18.06.0-ce`
+- `18.09.3-ce`
```

Comment:
We've updated this one to drop the `-ce` suffix, because docker started dropping this since `18.09.0`.